app.py

```diff
@@ -43,6 +45,20 @@
 app.secret_key = skey
 
 
+requests.packages.urllib3.util.ssl_.DEFAULT_CIPHERS += "HIGH:!DH:!aNULL"
+try:
+    requests.packages.urllib3.contrib.pyopenssl.DEFAULT_SSL_CIPHER_LIST += (
+        "HIGH:!DH:!aNULL"
```

Comment:
Please could you comment what this does? is it a testing environment?

Reply:
Added comment in latest push. I hope it helps

Reply:
Thanks so much for the comment, super informative and saves some stack overflow trawling.